tools/pot/openvino/tools/pot/app/argparser.py

```diff
@@ -47,6 +47,13 @@ def get_common_argument_parser():
         type=str,
         help='Model name. Applicable only when -q option is used.')
 
+    parser.add_argument(
+        '--engine',
+        default='accuracy_checker',
+        choices=['accuracy_checker', 'data_free', 'simplified'],
```

Comment:
we need to expose the Simplified mode through the command line as well because now it is absent. But let consider doing this in a separate PR.